generate/templates/api_doc.handlebars

```diff
@@ -34,12 +34,7 @@ import (
 )
 
 func main() {
-	client, err := vault.New(
-		vault.WithAddress("http://127.0.0.1:8200"),
-	)
-	if err != nil {
-		log.Fatal(err)
-	}
+	client := // a suitably configured *vault.Client (see top-level README.md)
```

Comment:
```suggestion
	client := vault.New(/* params */) // a suitably configured *vault.Client (see top-level README.md)
```

Reply:
Switched to a comment for the auth section instead.